### PR TITLE
remove fortify, add api  register and login

### PR DESCRIPTION
**Pull request by brisitech**

### CODE REVIEW

File: backend/app/Http/Controllers/API/RegisterController.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function register(Request $request): JsonResponse
+    {
+        $validator = Validator::make($request->all(), [
+            'name' => 'required',
+            'email' => 'required|email',
+            'password' => 'required',
+            'c_password' => 'required|same:password',
```
Comment (sovsky) [inline backend/app/Http/Controllers/API/RegisterController.php]: confirm_password zamiast c_password

File: backend/app/Http/Controllers/API/RegisterController.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+class RegisterController extends BaseController
+{
+    /**
+     * Register api
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function register(Request $request): JsonResponse
+    {
+        $validator = Validator::make($request->all(), [
```
Comment (sovsky) [inline backend/app/Http/Controllers/API/RegisterController.php:20]: brak city, age, phone_number

**Review by sovsky — APPROVED** [review]

File: backend/app/Http/Controllers/API/RegisterController.php (hunk 1 above)
Comment (brisitech) [reply backend/app/Http/Controllers/API/RegisterController.php]: I changed

File: backend/app/Http/Controllers/API/RegisterController.php (hunk 2 above)
Comment (brisitech) [reply backend/app/Http/Controllers/API/RegisterController.php:20]: I added

**Comment by brisitech** [comment]
I added error handling for each field in RegisterController and the age, city and phone_number fields for validation

**Review by sovsky — APPROVED** [review]

**Pull request merged**